src/Main.java

```diff
@@ -50,6 +50,6 @@ public static void main(String[] args) {
         var people3 = people + people2;
         System.out.println(people3);
         var hours1 = general / people3;
```

Comment:
здесь нужно не делить, а умножать, потому что чтобы узнать, сколько часов работы может быть поделено между сотрудниками, надо умножить кол-во сотрудников на hours 
```suggestion
        var hours1 = people3 * hours;
```